zulipterminal/helper.py

```diff
@@ -101,6 +107,14 @@ class UnreadCounts(TypedDict):
 }
 
 
+STATE_ICON = {
```

Comment:
Should we call it `STATE_ICONS` instead?

Reply:
Hmm. Seems more apt :+1:

Reply:
When naming like this, it's good to think of how it's used - `STATE_ICONS['active']` suggests there are more than one active one? Generally I err towards the non-plural for this reason.

Also the values are icons, but the keys aren't - so as always, naming is tricky to get perfect!

This relates application state values to representation, a little like `edit_mode_captions`, so perhaps it belongs in with that, though I'm not keen on `helper.py` for that.

Reply:
Just to resolve this discussion, I've migrated the `STATE_ICON` to `ui_mappings.py` as added to main recently via #1013.